samples/csharp/getting-started/BinaryClassification_HeartDiseasePrediction/HeartDiseasePrediction-Solution/DataStructure/HeartPrediction.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.ML.Data;
+
+namespace HeartDiseasePredictionConsoleApp.DataStructure
+{
+    public class HeartPrediction
+    {
+        // ColumnName attribute is used to change the column name from
+        // its default value, which is the name of the field.
+        [ColumnName("PredictedLabel")]
+        public bool Prediction { get; set; }
```

Comment:
No need to specify these as properties. You can specify them as fields. because we don't set anything in **Prediction** field.
```suggestion
[ColumnName("PredictedLabel")]
        public bool Prediction;
```